manual/src/tutorials/lablexamples.etex

```diff
@@ -192,7 +192,7 @@ received function is really applied to an argument.
 \section{s:label-suggestions}{Suggestions for labeling}
 
 Like for names, choosing labels for functions is not an easy task. A
-good labeling is a labeling which
+good label is one which
```

Comment:
I disagree with the `label` -> `labeling` change: the `labeling` version emphasizes that a good choice of label for an argument depends on the choices made for every other arguments. It is the labeling, in other words the set of labels for all arguments, which is a good fit for a function.

Reply:
Makes sense, updated in most recent commit.